src/tcompiler.cpp

```diff
@@ -1304,7 +1363,13 @@ struct FunctionEmitter {
                         }
                         CU->fpm->run(*scc[i]);
                         VERBOSE_ONLY(T) {
+                            #if LLVM_VERSION < 38
```

Comment:
This pattern occurs in a few places and should probably be encapsulated in a function

Reply:
Done